MIGRATION-v8.norg

```diff
@@ -149,17 +149,60 @@ version: 1.1.1
     Run `locate lua.h` and find the most relevant file.
     Pass the dirname to `luarocks_build_args` as below.
 
-    If you are on NixOS, a workaround is (ofc change the path to match your system)
-    @code lua
-    {
-      "vhyrro/luarocks.nvim",
-      opts = {
-        luarocks_build_args = {
-          "--with-lua-include=/nix/store/98blcb69q9qy0k279xjk10lcmfwnd4rg-luajit-2.1.1693350652/include",
-        },
-      },
-    }
-    @end
+**** NixOS
+     If you are on NixOS, there are a few ways to do things, as always.
```

Comment:
Please add a blank like after the header title since titles can be a paragraph from v3 parser.

https://github.com/boltlessengineer/tree-sitter-norg3-pr1/blob/1fa17fe9d2f8ee2ef21cebf2641173a444c2212c/CHANGELOG.norg#L28-L38

Reply:
I do not like that at all... but yes will do